subsys/net/lib/download_client/src/download_client.c

```diff
@@ -493,7 +493,6 @@ static int reconnect(struct download_client *dl)
 		err = close(dl->fd);
 		if (err) {
 			LOG_DBG("disconnect failed, %d", err);
-			return err;
```

Comment:
Just wondering if this change is intentional and necessary? I guess the original idea has been that if `close()` fails, reconnect is not tried again.

Reply:
It is intentional.
When `close()` fails, there is nothing else to do. But it does not mean that reconnect should not be tried again.